components/badge/badge.html

```diff
@@ -0,0 +1,229 @@
+            <a
+              class="primary-color cursor-pointer text-dec-none link-active"
+              href="/components/badge/badge.html"
+              >Badge</a
+            >
+          </li>
+          <li class="padding-lg">
+            <a
+              class="primary-color cursor-pointer text-dec-none"
+              href="/components/card/card.html"
+              >Card</a
+            >
+          </li>
+          <li class="padding-lg">
+            <a
+              class="primary-color cursor-pointer text-dec-none"
+              href="/components/image/image.html"
+              >Image</a
+            >
+          </li>
+          <li class="padding-lg">
+            <a
+              class="primary-color cursor-pointer text-dec-none"
+              href="/components/input/input.html"
+              >Input</a
+            >
+          </li>
+          <li class="padding-lg">
+            <a
+              class="primary-color cursor-pointer text-dec-none"
+              href="/components/navigation/navigation.html"
+              >Navigation</a
+            >
+          </li>
+          <li class="padding-lg">
+            <a
+              class="primary-color cursor-pointer text-dec-none"
+              href="/components/list/list.html"
+              >Lists</a
+            >
+          </li>
+          <li class="padding-lg">
+            <a
+              class="primary-color cursor-pointer text-dec-none"
+              href="/components/typography/typography.html"
+              >Typography</a
+            >
+          </li>
+        </ul>
+      </aside>
+      <section class="components">
+        <h1 class="h1 text-wt-bold">Badges</h1>
+        <p>
+          Badges are used to indicate notifications or user activity status.
+        </p>
+        <hr class="hr" />
+        <h3>Usage of status badges</h3>
+        <p>
+          We can have badges on avatars showing the user's activity status. You
+          can use <span class="code">badge-online</span>,
+          <span class="code">badge-away</span>,
+          <span class="code">badge-busy</span>, or
+          <span class="code">badge-offline</span> to indicate user status, along
+          with generic class <span class="code">badge</span>. Badge can also be
+          had in various sizes according to avatar size; use class
+          <span class="code">badge-lg</span>,
+          <span class="code">badge-md</span>,
+          <span class="code">badge-sm</span>, or
+          <span class="code">badge-xs</span>.
+        </p>
+        <hr class="hr" />
+        <div class="badge-wrapper">
+          <div class="avatar avatar-lg badge-container">
+            <img
+              class="img-responsive img-round"
+              src="/assets/images/avatar.jpg"
+              alt="avatar-large"
+            />
+            <span class="badge badge-online badge-lg"></span>
+          </div>
+          <div class="avatar avatar-md badge-container">
```

Comment:
If possible try using postion:relative in the avatar class only instead of badge-container, as suggested by MA try minimalizing the number of classes

Reply:
Will see if this is possible without much refactoring.